server/auth/types/openid/openid_auth.ts

```diff
@@ -119,21 +125,42 @@ export class OpenIdAuthentication extends AuthenticationType {
   }
 
   private createWreckClient(): typeof wreck {
-    const wreckHttpsOption: WreckHttpsOptions = {};
     if (this.config.openid?.root_ca) {
-      wreckHttpsOption.ca = [fs.readFileSync(this.config.openid.root_ca)];
+      this.wreckHttpsOption.ca = [fs.readFileSync(this.config.openid.root_ca)];
+      this.logger.debug(`Using CA Cert: ${this.config.openid.root_ca}`);
+    }
+    if (this.config.openid?.pfx) {
```

Comment:
Should this be `else if`? Is it redundant to configure `root_ca` and `pfx`?

Reply:
I don't believe it is redundant. In this case, the root_ca contains the trusted CA certificate(s) that are chainable to the IdP's cert. The pfx would be provided as the client certificate in lieu of the pem formatted private key and public key.

Reply:
You're right. Those options can both be set according to the close in [node](https://github.com/nodejs/node/blob/v21.2.0/lib/https.js#L236-L258). The node [documentation website](https://nodejs.org/api/https.html#new-agentoptions) does not specify what options are configurable for the HTTPS Agent.